fhir-core/src/main/java/com/ibm/fhir/core/util/handler/IPHandler.java

```diff
@@ -44,9 +44,13 @@ private static String determineIP() {
         try {
             ip = Inet4Address.getLocalHost().getHostAddress();
         } catch (UnknownHostException e) {
-            log.log(Level.SEVERE, "Failure acquiring local host IP address", e);
+            // Defaulting to the localhost, as the machine's name is not registered in the local lookup/dns.
+            log.log(Level.SEVERE, "Failure acquiring local host IP address - be sure the name is registered in the name resolution service", e);
+            ip = "127.0.0.1";
         } catch (NumberFormatException e) {
-            log.log(Level.SEVERE, "Failure parsing local host IP address '" + ip + "'", e);
+            // Defaulting to the localhost, as the machine's name is not registered in the local lookup/dns.
+            log.log(Level.SEVERE, "Failure parsing local host IP address '" + ip + "' - be sure the name is registered in the name resolution service" , e);
```

Comment:
Won't `ip` always be null here?  Maybe just combine the two catches?

Reply:
Yes, you are right, I merged the two